app/components/jobseekers/school_group_overview_component.html.haml

```diff
@@ -17,7 +17,7 @@
         %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
           = t('school_groups.email')
         %td.govuk-table__cell
-          -# TODO: Unable to get email from GIAS
+          = @vacancy.contact_email
```

Comment:
It should probably be this (like how it is for School listings)...

```
- if @vacancy.contact_email.present?
    %tr.govuk-table__row
      %td.govuk-table__cell{ class: "govuk-!-font-weight-bold" }
        = t('jobs.contact_email')
      %td.govuk-table__cell
        = mail_to @vacancy.contact_email, @vacancy.contact_email, class: 'govuk-link link-wrap', subject: t('jobs.contact_email_subject', job: @vacancy.job_title), body: t('jobs.contact_email_body', url: job_url(@vacancy))
```

Reply:
Thanks :)